azure-pipelines.yml

```
@@ -461,6 +461,16 @@ stages:
               ROR_TASK: integration_es810x
             IT_es80x:
               ROR_TASK: integration_es80x
+            IT_es80x:
```

YAMLlint (1.38.0)

[error] duplication of key "IT_es80x" in mapping

(key-duplicates)